README.md

```diff
@@ -37,11 +37,12 @@
 
 # Getting Started with nOS
 
-The standalone client installers can be found [here](https://github.com/nos/client/releases).
+The standalone client installers can be found 
+[here](https://github.com/nos/client/releases).
 To build manually, the client can be cloned from GitHub and run using the development steps below.
 
 ### Commands
-```
+```c
```

Comment:
This isn't C, so maybe `bash` or `shell` would be more appropriate?

Reply:
Indeed, this isn't C, but git markup bash is shit and doesn't show anything. C (and Cpp and derived languages, e.g. Java and Scala) will do the `//` correctly, but bash never helps for the build instructions. Actually, I usually go for haskell when I don't know, because that language always has nice highlighting. I think it's because of the pattern matching feature that are often tied to functional languages. 
Well, choose either aestetics: Using the right language flag or using the right color highlighting in practice. Sadly it doesn't seem to work to have both. 
Not sure if shell exist or if it does what you want - let's try. I don't think it will.

Reply:
Just checked: Both bash and shell will give you the opposite of what you want, namely highlighting a bunch of words in the comments.

Reply:
This is not the proper way to do it.

Best way is to indeed use `bash` or `sh` as highlighting target and use # as a comment identifier, so please change it to that.

Example:
```sh
# Cloning from Github
git clone https://github.com/nos/client.git

# Install dependencies and launch the nOS client
# This is used to start developing on the nOS client
yarn install && yarn start

# Testing command
yarn test

# Testing with debug (repl) command
yarn test:debug

# Distribution command
yarn dist
```